### DISCUSSION TITLE
Grove Offline Voice Recognition sensor | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1104] [Q&A]
# Grove Offline Voice Recognition sensor | Seeed Studio Wiki

Getting started for the Grove Offline Voice Recognition sensor.

https://wiki.seeedstudio.com/Grove-Offline-Voice-Recognition/

**Comment by steveastrouk** [comment]
Can it be trained with another vocabulary ?

**Comment by kaloprojects** [comment]
2 questions .. does someone know ?
- how can i modify the 'wakeup word' .. or is the "Hello Pudding" hard coded and can't be modified ?
- how to use own commands, can i edit the file 'commandBase.h' ?

Thx in prepare!
